sympy/utilities/tests/test_autowrap.py

```diff
@@ -181,11 +209,17 @@ def test_cython_wrapper_compile_flags():
 """ % {'num': CodeWrapper._module_counter}
 
     code_gen._need_numpy = True
-    code_gen._prepare_files(routine, build_dir=temp_dir)
+    code_gen._prepare_files(routine, build_dir=temp_dir, pyx_dir=temp_dir)
```

Comment:
Why can't the `build_dir` be used for the pyx files? Not clear why an extra directory needs to be specified.